CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+
+For a comprehensive guide on how to write small PRs, see this [guide](https://github.com/google/eng-practices/blob/master/review/developer/small-cls.md)
+
+## Contributor License Agreement (CLA)
+
+In order to accept your pull request, we need you to submit a CLA. You only need to do this once. Simply reply to your first pull request with `I have read the CLA Document and I hereby sign the CLA`. Your signature will be recorded automatically in the `cla` branch.
+
+## Code Reviews
+
+A code review is a process where someone other than the author(s) of a piece of code examines that code. Code committed to the codebase is both the responsibility of the author and the reviewer. Code reviews should look at:
+
+-   Design: Is the code well-designed and appropriate for your system?
+-   Functionality: Does the code behave as the author likely intended? Is the way the code behaves good for its users?
+-   Complexity: Could the code be made simpler? Would another developer be able to easily understand and use this code when they come across it in the future? Can multiple
+-   Tests: Does the code have correct and well-designed automated tests?
+-   Naming: Did the developer choose clear names for variables, classes, methods, etc.?
+-   Comments: Are the comments clear and useful?
+-   Style: Does the code follow our style guides? Note, in most cases, style nits should be avoided and be enforced entirely by automated tooling. However some stylistic decisions can be discussed if it impacts readability and complexity.
+-   Documentation: Did the developer also update relevant documentation?
+
+All of the above are grounds for a reviewer to request changes in a PR. Consensus should be reached to the best of the abilities of the author and reviewer. However, if consensus cannot be reached between the two parties, the review should be escalated to the technical lead.
+
+### Code Review Conduct
+
+-   Be kind.
+-   Explain your reasoning.
+-   Balance giving explicit directions with just pointing out problems and letting the developer decide.
+-   Encourage developers to simplify code or add code comments instead of just explaining the complexity to you.
+
+Remember that you are both on the same team and are working towards the same goal. The goal is not to make the code perfect, but to make it better. Make sure your comments are constructive and actionable. Treat the code diff as an non-personal artifact under inspection rather than a personal creation. For a comprehensive guide on how to provide effective PR feedback, see this [guide](https://google.github.io/eng-practices/review/reviewer/comments.html).
```

Comment:
```suggestion
Remember that you are both on the same team and are working towards the same goal. The goal is not to make the code perfect, but to make it better. Make sure your review comments are constructive and actionable. Treat the code contribution as a communal artifact under construction rather than a personal creation under evaluation. For a comprehensive guide on how to provide effective PR feedback, see this [guide](https://google.github.io/eng-practices/review/reviewer/comments.html).
```